lib/possible_bipartition.rb

```diff
@@ -1,4 +1,39 @@
-
+# Time complexity:O(n) - visiting each node in the graph once, searching in sets is O(1)
+# Space complexity: O(n) - the sets can be filled up based on the content of the dislikes
+require 'set'
 def possible_bipartition(dislikes)
```

Comment:
👍 For time complexity you're also checking every edge for every node so this is O(N + E)